proc-macros/tests/ui/correct/alias_doesnt_use_namespace.rs

```diff
@@ -1,12 +1,12 @@
-use jsonrpsee::{proc_macros::rpc, core::RpcResult};
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
 
 #[rpc(client, server, namespace = "myapi")]
 pub trait Rpc {
 	/// Alias doesn't use the namespace so not duplicated.
 	#[method(name = "getTemp", aliases = ["getTemp"])]
 	async fn async_method(&self, param_a: u8, param_b: String) -> RpcResult<u16>;
 
-	#[subscription(name = "getFood", item = String, aliases = ["getFood"], unsubscribe_aliases = ["unsubscribegetFood"])]
+	#[subscription(name = "subscribeGetFood", item = String, aliases = ["getFood"], unsubscribe_aliases = ["unsubscribegetFood"])]
```

Comment:
Let's see if I have this right. The methods generated here are:
- `myapi_subscribeGetFood` (auto)
- `getFood` (alias)
- `myapi_unsubscribeGetFood` (auto)
- `unsubscribegetFood` (alias)

Did I get that right?